spec/requests/users_spec.rb

```diff
@@ -3,15 +3,30 @@
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/users/index'
+      get '/users/'
       expect(response).to have_http_status(:success)
     end
+
+    it 'returns http success' do
+      get '/users/:id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should include correct placeholder' do
+      get '/users/:id'
+      expect(response.body).to include('Here are the users')
+    end
   end
 
   describe 'GET /show' do
     it 'returns http success' do
-      get '/users/show'
+      get '/users/:id'
       expect(response).to have_http_status(:success)
     end
```

Comment:
- [ ] Only one of your tests is passing. Kindly look into that and resolve it. I had to create a user and reconfigure the routing for some of the tests to start passing at all (`:id` in `users/:id` does not really exist, it needs to represent an actual record), like so:

  ```
  require 'rails_helper'

  RSpec.describe 'Users', type: :request do
    let!(:user) { User.create(name: "My name", bio: "my bio", posts_counter: 0) }
    describe 'GET /index' do
      it 'returns http success' do
        get '/users/'
        expect(response).to have_http_status(:success)
      end
  
      it 'should include correct placeholder' do
        get "/users/"
        expect(response).to render_template(:index)
      end
    end
  
    describe 'GET /show' do
      it 'returns http success' do
        get "/users/#{user.id}"
        expect(response).to have_http_status(:success)
      end
  ...
  ```

  ![fuff](https://user-images.githubusercontent.com/36542986/205866230-910d74df-67df-4960-bcdc-309b0ea1100d.jpg)